packages/components/src/CreditCard/Menu/Item.purs

```diff
@@ -1,55 +1,61 @@
 module KSF.CreditCard.Menu.Item where
 
-import Prelude (Unit, ($), (<>), show)
+import Prelude (Unit, ($), (<>), pure, show)
```

Comment:
Let's just remove the import list from this instead. The PureScript compiler seems to like to show a spurious warning about Prelude when there's any other warning as well but it's usually meant for plain importing.